cmd/peer-rest-client-target.go

```diff
@@ -31,6 +31,11 @@ func (target *PeerRESTClientTarget) ID() event.TargetID {
 	return target.id
 }
 
+// IsActive - Return true if target is up and active
+func (target *PeerRESTClientTarget) IsActive() (bool, error) {
```

Comment:
We could add a comment saying that this function does nothing and we added just for interface compatibility

Reply:
done.